### DISCUSSION TITLE
Reviews for Anime / Manga / Manwha

### DISCUSSION THREAD

**Discussion by KathEllieAelo** [subrotokumar/kurumi#8] [Ideas]
hi can u please add a reviews panel for anime / manga / manwha, like what is the review of that kind of anime in anilist, so i can read some of it and be able to know if its worth watching/reading

**Comment by subrotokumar** [comment]
Hi, thanks for your suggestion. A reviews panel for anime/manga/manhwa sounds like a good idea, as it would help in finding out what other people think of the shows or books you are interested in. I will consider adding this feature in the future, but I can't promise anything right now. I appreciate your feedback and hope you enjoy using our app.
